test/spec/protractor_sync_sherman_test.ts

```diff
@@ -0,0 +1,50 @@
+//TODO:  This is a temporary test file to be removed before merged.
+//TODO: The contents of this file should move ino README.md when ready
+import * as ab from 'asyncblock';
+
+import * as protractorSync from '../../app/index';
+import { browserSync, elementSync, polledExpect} from '../../app/index';
+
+protractorSync.configure({ implicitWaitMs: 500 });
+
+function createTest(fn: Function, errorMsg?: string) {
+  return (done: Function) => {
+    ab(() => {
+      fn();
+    }, (err: any) => {
+      if (errorMsg) {
+        expect(err.message).toEqual(errorMsg);
+      } else {
+        expect(err && err.stack || err || undefined).toBeUndefined();
+      }
+      done();
+    });
+  };
+}
+
+//TODO:  Ideally we have a test that highlights some of protractor-sync's awesome functionality.
+// I had an initial thought to emulate the subject matter in https://github.com/blackboard/protractor-sync/issues/62
+// but could be swayed to write a test against a different Web application
+//NOTE:  I was also uploaded on Chrome 65 and sendKeys was yielding this error:
+//    - Expected 'WebDriverError: unknown error: call function result missing 'value'
+//    - Updating Chromedriver to 2.36 solved this issue.
+describe('Google Translate', () => {
+  const googleTranslateUrl = 'https://translate.google.com/';
+
+  it('does not show the clear button when no text exists in the source field', createTest(() => {
+    browserSync.get(googleTranslateUrl);
+
+    const rootElement = elementSync.findElement('#gt-src-c');
+    browserSync.get(googleTranslateUrl);
+    polledExpect(() => rootElement.findElement('.clear-button').isDisplayed()).toEqual(false);
+  }));
+
+  it('does show the clear button after entering text in the source field', createTest(() => {
+    const expectedText = 'merci';
+    browserSync.get(googleTranslateUrl);
+
+    const rootElement = elementSync.findElement('#gt-src-c');
+    rootElement.findVisible('textarea#source').clear().sendKeys(expectedText);
```

Comment:
ah, I can send whatever we want here.  I'll follow up.  This was leftover from before.